scripts/qa_crowbarsetup.sh

```diff
@@ -3698,6 +3698,9 @@ function oncontroller_testsetup
 {
     . .openrc
     oncontroller_prepare_functional_tests
+    if [[ $hacloud = 1 ]] ; then
+        crm_mon --failcounts -1 | grep "fail-count=" && complain 34 "Cluster resources' failures detected"
```

Comment:
@vmoravec Do you mean you *weren't* expecting to catch all cases?

@dirkmueller @vmoravec I'm fine with starting low as long as we don't forget to raise the bar higher later on :)